### PR TITLE
revised log watching to use tail implementation

### PR DESCRIPTION
**Pull request by nolram88**
swapped to commons io tail implementation. should save a lot of memory and can track without polling.

### CODE REVIEW

**Comment by Pastillage** [comment]
This looks nice, gonna test it later when I'm home

**Comment by Pastillage** [comment]
Right now this code doesn't work, you need to change:
public static final String gameLogFile = PATH + "\\Client.txt";
to
public static final String gameLogFile = PATH;

Also please make this PR go to the 1.1 Branch.

Other than that I like it a lot and it works really well.

File: src/main/java/Data/ClientParser.java [hunk 1]
```diff
@@ -1,137 +1,58 @@
...
-            try
-            {
-                Thread.sleep(Settings.getPollingRate());
-            } catch (InterruptedException e)
-            {
-                e.printStackTrace();
-            }
-        }
-    }
+    public static final String gameLogFile = PATH + "\\Client.txt";
```
Comment (Pastillage) [inline src/main/java/Data/ClientParser.java:19]: Remove: + "\\Client.txt" in order for this to work.

Comment (nolram88) [reply src/main/java/Data/ClientParser.java:19]: oh right I wanted to suggest pointing it at the folder instead of directly at the file.

Comment (Pastillage) [reply src/main/java/Data/ClientParser.java:19]: Also not too bad of a way too do it, but I feel like forcing a user to select the Client.txt is a safer way of making sure the Client.txt is selected properly.

**Pull request closed without merging**